skills_ml/algorithms/embedding/models.py

```diff
@@ -0,0 +1,137 @@
+"""Embedding model class inherited the interface from gensim"""
+from skills_ml.storage import Store, FSStore
+from skills_ml.algorithms.embedding.base import Base2VecModel
+
+from gensim.models import Doc2Vec, Word2Vec
+
+import numpy as np
+import pickle
+import logging
+
+class Word2VecModel(Word2Vec, Base2VecModel):
+    """The Word2VecModel Object is a base object which specifies which word-embeding model.
+
+    Example:
+    ```
+    from skills_ml.algorithms.embedding.base import Word2VecModel
+
+    word2vec_model = Word2VecModel()
+    ```
+    """
+    def __init__(self, storage=None, *args, **kwargs):
+        """
+        Attributes:
+            storage (:obj: `skills_ml.Store`): skills_ml Store object
+            model (:obj: `gensim.models.doc2vec.Doc2Vec`): gensim doc2vec model.
+        """
+        super().__init__(*args, **kwargs)
+        self._storage = FSStore() if storage is None else storage
```

Comment:
Both of the subclasses appear to be bypassing the Base2VecModel mixin's storage init. This makes sense on the surface, as super only works for one base class and clearly the gensim class is the winner in both cases. I think you should be able to just call the specific `__init__` functions directly though, to be clear about how the kwargs are being partitioned between the base class and the mixin.

Example here (the accepted answer):

https://stackoverflow.com/questions/26927571/multiple-inheritance-in-python3-with-different-signatures

You can use kwargs.pop to make this more clear. I haven't tested this, but somehting like

```
Base2VecModel.__init__(self, storage=kwargs.pop('storage', None))
Word2Vec.__init__(self, *args, **kwargs)
```

might work.